stl/inc/charconv

```diff
@@ -448,7 +448,9 @@ _NODISCARD inline uint32_t _Bit_scan_reverse(const _Big_integer_flt& _Xval) noex
 
     unsigned long _Index; // Intentionally uninitialized for better codegen
 
-    _BitScanReverse(&_Index, _Xval._Mydata[_Bx]); // assumes _Xval._Mydata[_Bx] != 0
+    // assumes _Xval._Mydata[_Bx] != 0

```

Comment:
I don't think we need to repeat this here, on 447, and on 453. I suggest moving the `_STL_INTERNAL_CHECK` down to immediately precede the call to `_BitScanReverse`, and removing the other two occurrences.